app/assets/stylesheets/components/_border.scss

```diff
@@ -9,3 +9,7 @@
 .rounded-lg { border-radius: $border-radius-lg; }
 .rounded-xl { border-radius: $border-radius-xl; }
 .rounded-xxl { border-radius: $border-radius-xxl; }
+
+@media #{$breakpoint-sm} {
+  .sm-border-none { border: 0; }
```

Comment:
@jessieay haha, yeah that addition to the stylesheet is to allow a hortizonal border to be shown in the footer, but only on the mobile view. The padding comes from the addition of atomic css classes in the view like this: `.pt1.pb2.sm-pt0.sm-pb0` which adds padding to the top and bottom, but only on the mobile view.  `pt1` = `padding-top: 8px`, `pb2` = `padding-bottom: 16px`